lib/spotify_uri_bot/middleware/ignore_bot_message.ex

```diff
@@ -0,0 +1,15 @@
+defmodule SpotifyUriBot.Middleware.IgnoreBotMessage do
+  use ExGram.Middleware
+
+  alias ExGram.Cnt
+
+  def call(%Cnt{update: %{message: %{text: text}}} = cnt, _opts) do
+    if String.contains?(text, "🔗 URI: ") do
```

Comment:
Instead of adding the extra and mathing it outside, if you just want to ignore the message, you can do:

``` elixir
if String.contains?(text, "whatever") do
  cnt
else
  %{cnt | halted: true}
end
```

On the middleware.ex module we have `halt/1` but I'm reviewing the code and it does some weird things, so changing the `halted` term here is the best option 😂

Reply:
Very interesting! Thanks!